features/admin.remote-userstores.v1/components/edit/userstore-general-settings.tsx

```diff
@@ -517,354 +516,356 @@ export const UserStoreGeneralSettings: FunctionComponent<UserStoreGeneralSetting
 
                 // ATM, userstore operations run as an async task in the backend. Hence, The changes aren't
                 // applied at once. As a temp solution, a notification informing the delay is shown here.
                 // See https://github.com/wso2/product-is/issues/9767 for updates on the backend improvement.
                 // TODO: Remove delay notification once backend is fixed.
-                dispatch(addAlert<AlertInterface>({
-                    description: t("userstores:notifications.updateDelay.description"),
-                    level: AlertLevels.WARNING,
-                    message: t("userstores:notifications.updateDelay.message")
-                }));
+                dispatch(
+                    addAlert<AlertInterface>({
+                        description: t("userstores:notifications.updateDelay.description"),
+                        level: AlertLevels.WARNING,
+                        message: t("userstores:notifications.updateDelay.message")
+                    })
+                );
 
                 // Re-fetch the user store details
             })
-            .catch((error: { description: string, message: string }) => {
-                dispatch(addAlert<AlertInterface>({
-                    description: error?.description
-                        || t("userstores:notifications." +
-                            "updateUserstore.genericError.description"),
-                    level: AlertLevels.ERROR,
-                    message: error?.message || t("userstores:notifications." +
-                        "updateUserstore.genericError.message")
-                }));
+            .catch((error: { description: string; message: string }) => {
+                dispatch(
+                    addAlert<AlertInterface>({
+                        description:
+                            error?.description ||
+                            t("userstores:notifications." + "updateUserstore.genericError.description"),
+                        level: AlertLevels.ERROR,
+                        message:
+                            error?.message || t("userstores:notifications." + "updateUserstore.genericError.message")
+                    })
+                );
             });
     };
 
+    const renderLoadingSkeleton = (): ReactElement => {
+        return (
+            <div data-componentid={ `${componentId}-loading-skeleton` }>
+                <Skeleton component="h1" width={ "40%" } />
+                <Skeleton />
+                <br />
+                <Skeleton component="h1" width={ "40%" } />
+                <Skeleton />
+            </div>
+        );
+    };
+
+    const renderEmptyPlaceholder = (): ReactElement => {
+        return (
+            <EmptyPlaceholder
+                data-componentid={ `${componentId}-empty-placeholder` }
+                title="No Agents Connected"
+                subtitle={ [
+                    "There are no user store agent connections.",
```

Comment:
Shall we add i18n translations to this?

Reply:
Fixed 70f200f14255f037a46096205acd8f666919bb76